packages/website/src/layouts/default.astro

```diff
@@ -0,0 +1,66 @@
+---
+interface Cover {
```

Comment:
Hiermee moeten we zelf goed in sync houden of de types die we hier maken kloppen met de zod schema's. Ik stel voor deze types automatisch te laten genereren https://github.com/nl-design-system/gebruikersonderzoeken/pull/360